### PR TITLE
Issue #207: Collated a list of useful tools under "Resources" 

### PR DESCRIPTION
**Pull request by heidinett**
#207 Created tools section under "Resources" tab to collate a list index of useful tools

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -109,6 +109,8 @@ nav:
   - Working Groups:
     - working_groups/index.md
   - Resources:
+    - Data: resources/data.md
```
Comment (aidanheerdegen) [inline mkdocs.yml:112]: The navigation refers to a file `resources/data.md` that doesn't exist in the repository:

https://github.com/ACCESS-Hive/access-hive.github.io/tree/heidinett/issue207/docs/resources

**Comment by rbeucher** [comment]
I am not a big fan. The Hive is a collection of "ressources" already...

**Comment by aidanheerdegen** [comment]
> I am not a big fan. The Hive is a collection of "ressources" already...

It's a good point. We set up the original structure that maps to our activities, but then things pop up that aren't a good fit, e.g. the glossary links that @heidinett wanted to add. So we added "Resources" for this. Then it seemed that some other things fit better there than in the more NRI-centric categories. 

I recall that we had a discussion related to this when trying to decided between split sites (NRI/Community) and a single site, that there was a tension between how we would want to organise things from an NRI perspective versus what makes sense for the community.

So: we can rename "Resources" to "Miscellaneous" or "General", or we could have a wider conversation about the categories and organisation.

**Comment by rbeucher** [comment]
I agree there is a need for some reorganisation, especially in the Data/evaluation sections..

**Comment by aidanheerdegen** [comment]
> I agree there is a need for some reorganisation, especially in the Data/evaluation sections..

I've created an issue https://github.com/ACCESS-Hive/access-hive.github.io/issues/254

But in the mean time I think we should continue with "Resources".

**Pull request closed without merging**